compress.jl

```diff
@@ -0,0 +1,42 @@
+using NetCDF
+using NCDatasets,ZfpCompression,BSON,Statistics
+
+
+
+oldfile = "/Users/Minwoo/Desktop/emis/METDOT3D_160701.nc"
+
+#Compress function
```

Comment:
More detail: https://docs.julialang.org/en/v1/manual/documentation/#Functions-and-Methods